tests/osaka/eip7692_eof_v1/eip7620_eof_create/test_subcontainer_validation.py

```diff
@@ -832,8 +832,8 @@ def test_wide_container(eof_test: EOFTestFiller, width: int, exception: EOFExcep
                 ]
                 + 2 * [abort_sub_container],
                 expected_bytecode="""
-                ef0001010004020001000b03000200140014040000000080000436600060ff6000ec015000ef00010100
-                0402000100010400000000800000feef000101000402000100010400000000800000fe""",
+                ef0001010004020001000b03000200140014ff0000000080000436600060ff6000ec015000ef00010100
```

Comment:
@marioevz 
this is what I mean. one byte change in the format and we have to correct all the tests. 
in this case I think it is better we have a few unit tests for eof format builder. 
and disable expected_bytecode validation in most of EOF tests. unless EOF want to stay super secure and maintain it.

Reply:
I second this